purchases/src/test/java/com/revenuecat/purchases/amazon/AmazonBillingTest.kt

```diff
@@ -1117,6 +1116,31 @@ class AmazonBillingTest {
         assertThat(consent).isEqualTo(AmazonLWAConsentStatus.CONSENTED)
     }
 
+    @Test
+    fun `calling getUserData() in DefaultPurchasingServiceProvider sets LWA flag in builder`() {
+        setup()
+
+        mockkStatic(UserDataRequest::class)
+        mockkStatic(PurchasingService::class)
```

Comment:
Need to add a `tearDown` method to unmock these like:
```
@After
fun tearDown() {
    clearAllMocks()
}
```
Otherwise, the mockk will live through different tests.

Reply:
added 👍